server/routes/bookedtoday/unexpectedArrivals/noExistingRecordsFoundController.ts

```diff
@@ -1,9 +1,20 @@
 import type { RequestHandler } from 'express'
+import { State } from '../arrivals/state'
 
-export default class NoExistingRecordsFoundController {
+export default class NoMatchingRecordsFoundController {
```

Comment:
doesn't match the file name

Reply:
`no existing record` is probably better? - it matches the other `no existing record`page. 
The `no matching record` page refers to automatically matching

Reply:
ok